source/ref_packagers.rst

```diff
@@ -0,0 +1,462 @@
+        # You should provide a list of connections for a packager can get access to them.
+        targets: []
+    # Every packager saves its results into status field.
+    # Example of fields: docker image or python packager name.
+    status:
+        results:
+            - name: some_param
+              value: some_value
+
+.. note::
+
+    You can find an artifactName in the `status.artifactName` field of a model training entity.
+
+*********************
+Packagers management
+*********************
+
+Packagers can be managed using the following ways.
+
+Swagger UI
+----------
+
+ModelPackaging and PackagingIntegration are available on the Swagger UI at http://api-service/swagger/index.html URL.
+
+Odahu-flow CLI
+--------------
+
+:ref:`ref_odahuflowctl:Odahuflowctl` supports the Packagers API.
+You must be :ref:`logged in <ref_odahuflowctl:Login>` if you want to get access to the API.
+
+Getting all packaging in json format:
+
+.. code-block:: bash
+
+    odahuflowctl pack get --format json
+
+Getting the arguments of the packagers:
+
+.. code-block:: bash
+
+    odahuflowctl pack get --id tensorflow-cli -o 'jsonpath=[*].spec.arguments'
+
+* Creating of a packager from `pack.yaml` file:
+
+.. code-block:: bash
+
+    odahuflowctl pack create -f pack.yaml
+
+* All packager commands and documentation:
+
+.. code-block:: bash
+
+    odahuflowctl pack --help
+
+JupyterLab
+----------
+
+Odahu-flow provides the :ref:`int_jupyterlab_extension:JupyterLab extension` for interacting with Packagers API.
+
+********************************************
+Docker REST
+********************************************
+
+The Docker REST packager wraps an ML model into the REST service inside a Docker image.
+The resulting service can be used for point prediction thorough HTTP.
+
+The packager provides the following list of targets:
+
+.. csv-table::
+   :header: "Target Name", "Connection Types", "Required", "Description"
+   :widths: 20, 20, 10, 100
+
+   "docker-push", ":ref:`docker<ref_connections:Docker>`, :ref:`ecr<ref_connections:Amazon Elastic Container Registry>`", "True", "The packager will use the connection for pushing a Docker image result"
+   "docker-pull", ":ref:`docker<ref_connections:Docker>`, :ref:`ecr<ref_connections:Amazon Elastic Container Registry>`", "False", "The packager will use the connection for pulling a custom base Docker image"
+
+The packager provides the following list of arguments:
+
+.. csv-table::
+   :header: "Argument Name", "Type", "Default", "Required", "Description"
+   :widths: 20, 20, 20, 10, 100
+
+   "imageName", "string", "{{ Name }}-{{ Version }}:{{ RandomUUID }}", "False", "This option provides a way to specify the Docker image name. You can hardcode the full name or specify a template. Available template values: Name (Model Name), Version (Model Version), RandomUUID. Examples: myservice:123, {{ Name }}:{{ Version }}"
+   "port", "integer", "5000", "False", "Port to bind"
```

Comment:
params description says integer but you provide examples in quotes, is it correct ?

Reply:
![Screenshot from 2019-12-13 13-44-59](https://user-images.githubusercontent.com/6774206/70790600-dbf4f400-1dae-11ea-8fd3-94d17c27e5c8.png)
Yeap, the result table does not contain any quotes.